### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by bmazerolle**
Changes
---
- Add login controller testing
- Add admin controller testing
- Add file upload controller testing stub

Testing
---
- Covers all cases from 1 & 2 in the doc and passes as expected (This functionality is not broken)

### CODE REVIEW

File: src/test/java/com/uvic/venus/FileTests.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+@SpringBootTest
+public class FileTests {
+
+    @InjectMocks
+    FileController fileController;
+
+    @Mock
+    StorageService storageService;
+}
```
Comment (ACHultman) [inline src/test/java/com/uvic/venus/FileTests.java:17]: Any tests?

**Review by ACHultman — APPROVED** [review]
lgtm, no expert though

File: src/main/java/com/uvic/venus/controller/AdminController.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
 import org.springframework.security.core.GrantedAuthority;
 import org.springframework.security.core.authority.SimpleGrantedAuthority;
 import org.springframework.security.core.userdetails.User;
-import org.springframework.security.core.userdetails.UserDetails;;
+import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.security.provisioning.JdbcUserDetailsManager;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
```
Comment (ACHultman) [inline src/main/java/com/uvic/venus/controller/AdminController.java:17]: may be out of scope but some of these imports may be unused

**Review by ACHultman — APPROVED** [review]

File: src/main/java/com/uvic/venus/controller/AdminController.java (hunk 2 above)
Comment (bmazerolle) [reply src/main/java/com/uvic/venus/controller/AdminController.java]: Yea good point, didn't review those. Fixed

**Pull request merged**